network/tls_wrapper.go

```diff
@@ -108,14 +137,28 @@ func NewTLSConfig(serverName string, caPath, keyPath, crtPath string, authType t
 		}
 		certificates = append(certificates, cer)
 	}
+
+	verifyPeerCertificate := func(rawCerts [][]byte, verifiedChains [][]*x509.Certificate) error {
+		err := certVerifier.Verify(rawCerts, verifiedChains)
+
+		if err != nil {
```

Comment:
can we log it by one line without extra if statements like `log.WithErorr(err).WithField("status", err == nil).Debugln("verifyPeerCertificate")` ? and return error
Then we will have log message like:
```
time=2020-10-10T10:10:10 verifyPeerCertificate status=true error=nil

or

time=2020-10-10T10:10:10 verifyPeerCertificate status=false error=somethingWrong
```